research/picai/nnunet_scripts/transfer_train.py

```diff
@@ -0,0 +1,277 @@
+"""Automates simple transfer learning for nnUNetv2
```

Comment:
I think this is good documentation! Perhaps you can move this to the README.md in nnunet_scripts in a seperate section. Pretty much the information here and maybe some high level information on calling the script and things to watch out for. I can make a small PR in the future to cater the other content to be more general and put the vector specific information in a different file.